### PR TITLE
Update readme based on new website content

### PR DESCRIPTION
**Pull request by jlnhnng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,68 @@
 # k0smotron
 
-The Kubernetes control plane manager. Deploy and run Kubernetes control planes powered by [k0s](https://k0sproject.io) on any existing cluster.
+An open source control plane manager for unified cluster management.
+
+k0smotron allows you to unify your Kubernetes cluster management for an efficient use of resources. It’s designed for k0s.
```
Comment (jnummelin) [inline README.md:5]: A link to k0s would be nice.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,68 @@
...
-## Installation
+### Multi-cloud Cluster LCM
 
-```
-kubectl apply -f https://docs.k0smotron.io/stable/install.yaml
-```
+A multi-cloud strategy is essential these days. But managed Kubernetes offerings are often different in versioning or even the built-in tooling.
 
-## Creating a cluster
+With k0smotron, you can run the control plane management cluster in a public or private cloud provider of your choice and the worker nodes in different clouds. With this you have an approach to unified cluster management, providing one flavor on all clouds and saving the costs for the highly available Control Plane in the cloud.
```
Comment (jnummelin) [inline README.md:43]: I'd add that users can also get a homogenized central management over various clouds.

File: README.md [hunk 3]
```diff
@@ -1,71 +1,68 @@
...
-EOF
+```bash
+kubectl apply -f https://docs.k0smotron.io/stable/install.yaml
 ```
 
-## Creating cluster join tokens
+## Creating a cluster
 
-At the moment there isn't an automated way to gather one, you may obtain one by running 
+To create a cluster, you need to create a `Cluster` resource. The `spec` field is used for optional settings, so you can just pass `null` as the value.
```
Comment (jnummelin) [inline README.md:54]: Link to docs would be nice.

File: README.md [hunk 4]
```diff
@@ -1,71 +1,68 @@
...
+
+k0smotron allows you to unify your Kubernetes cluster management for an efficient use of resources. It’s designed for k0s.
 
 ## Features
 
-### Kubernetes-in-Kubernetes
+### Control Plane-as-a-Service
 
-k0smotron allows you to easily create and manage the clusters in an existing Kubernetes cluster. This allows unparalled scalability and flexibility when you have to work with many clusters. It allows truly homogenous setup for all control planes and thus eases the maintenance burden.
+k0smotron streamlines k0s control plane creation and management within your management cluster, reducing traditional operational overhead. k0smotron encapsulates the control plane service as a pods (and other Kubernetes constructs) and provides an intuitive approach to cluster lifecycle management through ClusterAPI integration.
```
Comment (jnummelin) [inline README.md:11]: > reducing traditional operational overhead

Overhead of what? Would be nice to explain a bit more what we mean with this.

File: README.md [hunk 5]
```diff
@@ -1,71 +1,68 @@
...
+### Enhanced High Availability
 
-Using k0smotron the clusters controlplane and workerplane are truly separated. The controlplane, running on an existing cluster has no direct networking connection to the workerplane. This is similar patter nhow all the major cloud providers separate the control and worker planes on the managed clusters. 
+k0smotron allows you to leverage the power of SQL databases as a data store for your control planes through Kine. This flexibility means you can choose from a variety of data storage operators like Postgres, MySQL, or your cloud provider's managed databases. This offers a robust, high-availability solution, eliminating the dependency on Etcd.
 
-### Bring your own workers
+### Bring Your Own Worker Nodes
 
-With k0smotron you can connect worker nodes from ANY infrastructure to your cluster control plane. 
+k0smotron prioritizes flexibility in the integration of worker nodes and allows easy connection or creation of nodes. This ensures node isolation and flexible scaling, minimizing interference with the control plane. k0smotron operates with or without the ClusterAPI, offering you the freedom to select your preferred operational mode.
```
Comment (jnummelin) [inline README.md:19]: I think it is important to mention that the worker nodes can be connected from ANY infra.

File: README.md [hunk 6]
```diff
@@ -1,71 +1,68 @@
...
+k0smotron streamlines k0s control plane creation and management within your management cluster, reducing traditional operational overhead. k0smotron encapsulates the control plane service as a pods (and other Kubernetes constructs) and provides an intuitive approach to cluster lifecycle management through ClusterAPI integration.
 
-### True control and worker plane separation
+### Enhanced High Availability
 
-Using k0smotron the clusters controlplane and workerplane are truly separated. The controlplane, running on an existing cluster has no direct networking connection to the workerplane. This is similar patter nhow all the major cloud providers separate the control and worker planes on the managed clusters. 
+k0smotron allows you to leverage the power of SQL databases as a data store for your control planes through Kine. This flexibility means you can choose from a variety of data storage operators like Postgres, MySQL, or your cloud provider's managed databases. This offers a robust, high-availability solution, eliminating the dependency on Etcd.
```
Comment (jnummelin) [inline README.md:15]: Yes, SQL DBs are one part of the story. But IMO more important is the fact that we can leverage Kubernetes "auto healing" etc. features.

File: README.md [hunk 7]
```diff
@@ -1,71 +1,68 @@
...
-
-k0smotron is an Kubernetes operator designed to manage the lifecycle of k0s control planes in a Kubernetes (any distro) cluster. By running the control plane on a k8s cluster we can enjoy and leverage the high availability and auto-healing functionalities of the underlying cluster, a.k.a Mothership.
+The k0smotron controller manager is a service that will be installed into an existing Kubernetes cluster. k0smotron will create and manage k0s control planes in the management cluster like workloads. It leverages the natural pattern of working with custom resources to manage the lifecycle of the k0s control planes. k0smotron will automatically create all the needed Kubernetes lower level resources, such as pods, configmaps, etc.
 
-TODO: Insert pic
+k0smotron is a service designed to manage the lifecycle of k0s control planes in a Kubernetes (any distro) cluster. By running the control plane on a k8s cluster we can enjoy and leverage the high availability and auto-healing functionalities of the management cluster.
```
Comment (jnummelin) [inline README.md:25]: IMO this and the paragraph above are quite much overlapping.

Also explain a bit how this integrates with ClusterAPI.

**Review by jnummelin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jlnhnng) [reply README.md:43]: Added "homogenized" to it.

File: README.md (hunk 4 above)
Comment (jlnhnng) [reply README.md:11]: Traditional operational overhead, like infrastructure management, installation and configuration, upgrades and patches, security monitoring and troubleshooting, backup and dr or high availability. Unfortunately, listing all of these will exceed the aligned lengths of the three listed features. Which of these operational tasks do you find most useful to list?

File: README.md (hunk 5 above)
Comment (jlnhnng) [reply README.md:19]: Added to the text.

File: README.md (hunk 6 above)
Comment (jlnhnng) [reply README.md:15]: Added to paragraph.

File: README.md (hunk 4 above)
Comment (jlnhnng) [reply README.md:11]: I've added 4 examples.

**Review by juanluisvaladas — CHANGES_REQUESTED** [review]
The changes look good to me but the git history is a bit messy, could you squash the commits into just one? Ping me on slack if you need some assistance with this.

**Review by juanluisvaladas — APPROVED** [review]

**Review by jnummelin — APPROVED** [review]

**Pull request merged**